03/app.js

```diff
@@ -1,5 +1,26 @@
 document.addEventListener('DOMContentLoaded', init);
 
+const btn = document.querySelector('button')
+const spanEl = document.querySelector('span')
+
+function getIp() {
+    const promise = fetch('https://api.ipify.org?format=json')
+    promise.then(resp => {
+        if (resp.ok) {
+            return resp.json()
```

Comment:
Tutaj zwracasz odpowiedź i ona powinna być przejęta czyli coś takiego:
```
const response =  promise.then(resp => { ...
```
wtedy można zrobić `response.then(...)`

Można to również skrótowo zapisać tj.
```
promise.then(resp => { ... } )
    .then( ... )  // tutaj jest tylko kropka nie ma zmiennej!
```

Natomiast Ty 2x "odpytujesz " `promise` i dlatego to nie działa.